lib/core/theme/app_theme.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import 'package:superformula_test/core/theme/app_colors.dart';
+
+class AppTheme {
```

Comment:
**question**: why did you create a class with a single `static` member? Would you just create a top-level `final` variable (without a class) otherwise? 🙂

Reply:
I did not think to just create a top-level variable, but that makes sense. I am more used to creating classes with static members. But your approach would be simpler!